src/main/resources/assets/angelica/lang/en_US.lang

```diff
@@ -53,6 +53,8 @@ sodium.options.use_entity_culling.name=Entity Culling
 sodium.options.use_entity_culling.tooltip=If enabled, entities determined not to be in any visible chunks will be skipped during rendering. This can help improve performance by avoiding the rendering of entities located underground or behind walls.
 sodium.options.use_particle_culling.name=Particle Culling
 sodium.options.use_particle_culling.tooltip=If enabled, only particles which are determined to be visible will be rendered. This can provide a significant improvement to frame rates when many particles are nearby.
+sodium.options.use_sodium_smooth_lighting.name=Sodium Smooth Lighting
+sodium.options.use_sodium_smooth_lighting.tooltip=If enabled, smooth lighting will be forced to use Sodium's lighting pipeline. This can provide more correct lighting when not using shaders, but is potentially slower.
```

Comment:
when using shaders*

Reply:
This actually doesn't effect shader lighting at all, I don't know why, but changing it seems to have zero impact when shaders are enabled.